www/master/components/dashboard/views/listPodsCards.html

```diff
@@ -43,7 +43,7 @@
               <md-grid-tile-footer>
 
                 <div class="pod-host">{{pod.currentState.host}}</div>
-                <div><a ng-href="/#/dashboard/pods/{{ pod.id }}"><h3>{{ pod.id }}</h3></a></div>
+                <div><a ng-href="#/dashboard/pods/{{ pod.id }}"><h3>{{ pod.id }}</h3></a></div>
```

Comment:
id is gone. now metadata.name

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api.md#v1beta3-conversion-tips